src/v2/Apps/Order/Utils/currencyUtils.ts

```diff
@@ -0,0 +1,11 @@
+export const appendCurrencySymbol = (
+  price: string | number | null | undefined,
+  currency: string
+): string | number | null | undefined => {
+  const pricePresent: boolean =
+    typeof price === "string" || typeof price == "number"
+  if (currency === "USD" && pricePresent) {
```

Comment:
I'm curious about the other currencies, shouldn't we support them as well?

Reply:
nvm this comment, I missed `we are not showing anything to indicate if the currency is US dollars, Hong Kong dollars, Australian dollars, etc.`

Reply:
Yes, also since this only touches BNMO works and we only support USD, EUR, and GBP this works for now as well. Although I agree we should have a better way of dealing with our currency types!